shared/common-adapters/list.native.js

```diff
@@ -32,6 +32,14 @@ class List extends PureComponent<Props<any>, void> {
     return (
       <View style={collapseStyles([styles.outerView, this.props.style])}>
         {/* need windowSize so iphone 6 doesn't have OOM issues */}
+        {/* We can use
+            initialScrollIndex={this.props.fixedHeight ? this.props.selectedIndex : undefined}
```

Comment:
Ideally we'd center the scroll view so that the selected wallet is visible, but that didn't work well with indices close to the end.